lib/ex_admin/ex_admin.ex

```diff
@@ -237,7 +210,7 @@ defmodule ExAdmin do
 
   @doc false
   def get_title_actions(%Plug.Conn{private: _private, path_info: path_info} = conn) do
-    defn = get_registered_by_controller_route(path_info)
+    defn = conn.assigns.defn
```

Comment:
I wonder if we should be cautions here and namespace the assigns. I just did this for the required abbr feature .I used `:ea_required`. So, what do you think of `:ea_defn`? BTW, I used `ea` as short for **e**x_**a**dmin.